cutten-web/src/components/UploadButtonV2.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+
+import {
+  gql
+} from 'apollo-boost';
+
+import {
+  Mutation
+} from 'react-apollo';
+
+import {
+  Button
+} from 'kingsbury/lib';
+
+interface IUploadButtonState {
+  file: File | null;
+}
+
+export const UPLOAD_FILE = gql`
+  mutation fuckyouu($file: Upload!) {
+    upload(file: $file)
+  }
+`;
+
+class UploadButtonV2 extends React.Component<any, IUploadButtonState> {
```

Comment:
Call this `UploadButton` if you are getting rid of the original one.

Reply:
Roger, roger